client/src/components/directory/UserDialog.jsx

```diff
@@ -301,9 +324,37 @@ class UserDialog extends Component {
             {subtitle}
           </DialogContentText>
           {
-            fields.map(infoField => (
-              this.renderInputComponent(infoField)
-            ))
+            fields.map(infoField => {
+              const { error } = this.state;
+              const { key, helperText } = infoField;
+              if (key === 'phone')
+                return (
+                  <InputMask
+                    key={key}
+                    mask="( 999 ) 999 - 9999"
```

Comment:
How does this mask look? I prefer to not have the circle brackets but a dash

Reply:
I can take out the brackets and add dashes back